pkg/plugins/builtin/email/README.md

```diff
@@ -0,0 +1,34 @@
+# `email` plugin
+
+This plugin send an email.
+
+## Configuration
+
+An action of type `email` requires the following kind of configuration:
+
+```yaml
+action:
+  type: email
+  configuration:
+    # mandatory, string
+    smtp_username: {{.config.smtp.username}}
+    # mandatory, string
+    smtp_password: {{.config.smtp.password}}
+    # optional, uint (default is 25)
+    smtp_port: {{.config.smtp.port}}
+    # mandatory, string
+    smtp_hostname: {{.config.smtp.hostname}}
+    # mandatory, string
+    from: foo@ovh.cloud
```

Comment:
ovh.cloud may belong to someone

RFC2606 specify domains suitable for documentation:
- example.com
- example.net
- example.org

https://tools.ietf.org/html/rfc2606#section-3